lib/node/runner/checker.go

```diff
@@ -6,8 +6,6 @@ import (
 	"encoding/json"
 	"io"
 
-	logging "github.com/inconshreveable/log15"
```

Comment:
@spikeekips I think this position is better because of code consistency.

Reply:
I also wanted, but I updated my ‘vim-go’ and then new `goimports` forced to rearrange like that :(

Reply:
Oh, then nevermind :)